src/routes/paths.ts

```diff
@@ -13,3 +13,37 @@ export const APPLICATION_LIST_PATH = WORKSPACE_PATH.extend(`applications`);
 export const APPLICATION_DETAILS_PATH = APPLICATION_LIST_PATH.extend(
   `:${RouterParams.applicationName}`,
 );
+
+export const APPLICATION_RELEASE_LIST_PATH = APPLICATION_DETAILS_PATH.extend(`releases`);
+
+export const APPLICATION_RELEASE_DETAILS_PATH = APPLICATION_RELEASE_LIST_PATH.extend(
+  `:${RouterParams.releaseName}`,
+);
+
+export const APPLICATION_SNAPSHOT_LIST_PATH = APPLICATION_DETAILS_PATH.extend(`snapshots`);
```

Comment:
Where is this route being used?

Reply:
It is used by ReleaseOverviewTab, so does pipeline related path. Let me remove them.